theme.html

```diff
@@ -0,0 +1,284 @@
+  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css">
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
+<script src="https://code.jquery.com/jquery-3.5.1.min.js"></script>
+<script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.1/dist/umd/popper.min.js"></script>
+<script src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/js/bootstrap.min.js"></script>
+<meta name="viewport" content="width=device-width, initial-scale=1">
+<style>
+  .bs-example{
+      margin: 20px;
+    }
+</style>
+<!-- <style>
+  .bottom {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+}
+</style> -->
+   <script>
+      function reloadFunction(event) {
+         if (event.persisted) {
+            console.log("The page was cached by the browser");
+            location.reload(true);
+         } else {
+            console.log("The page was NOT cached by the browser");
+         }
+      }
+   </script>
+</head>
+
+<body>
+   <div id="top-logo-placeholder" class="container">
+   </div>
+   <!-- Section for social media links starts -->
+   <div class="icon-bar">
+      <a href="https://www.facebook.com/FOSSEENMEICT/" class="facebook text-white" target="_blank"><i
+            class="fab fa-facebook-square"></i></a>
+      <a href="https://www.instagram.com/fossee_nmeict/" class="instagram" target="_blank"><i
+            class="fab fa-instagram"></i></a>
+      <a href="https://twitter.com/FOSSEE/" class="twitter" target="_blank"><i
+            class="fab fa-twitter-square"></i></a>
+      <a href="https://www.linkedin.com/company/fossee-iitb/" class="linkedin" target="_blank"><i
+            class="fab fa-linkedin"></i></a>
+   </div>
+   <!-- Section for social media links ends -->
+  <div class="box">
+   <div class="box-inner px-1">
+      This website will be offline for maintenance and updates from 01:30 AM to 02:00 AM IST.
+   </div>
+   </div>
+   <button onclick="topFunction()" id="myBtn" title="Go to top"><i class="fa fa-angle-up"
+         aria-hidden="true"></i></button>
+  <nav class="navbar navbar-expand-lg site-header sticky-top py-1">
+      <a class="navbar-brand" href="#"><span class="navbar-toggler-icon"></span></a>
+      <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent"
+         aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+         <span class="navbar-toggler-icon" style="color:#f50000"><i class="fas fa-align-justify"></i></span>
+      </button>
+      <div class="collapse navbar-collapse" id="navbarSupportedContent">
+         <ul class="navbar-nav mr-auto  mx-auto navbar-right">
+            <li class="nav-item active">
+               <a class="nav-link" href="#">Home <span class="sr-only">(current)</span></a>
+            </li>
+            <li class="nav-item">
```

Comment:
Update the nav links to index.html wherever required

Reply:
Done